src/components/Select/Select.tsx

```diff
@@ -16,7 +19,10 @@ export interface SelectProps extends DefaultProps {
 	options: SelectOption[];
 	id?: string;
 	disabled?: boolean;
+	loading?: boolean;
+	clearable?: boolean;
```

Comment:
Als we dan toch de react-select gaan gebruiken kunnen we dan niet beter extenden van die zijn props zodat alles bruikbaar is?

Reply:
bepaalde props hebben een andere naam en ik wilde vermijden dat we door alle client code gingen moeten gaan en die props updaten